packages/ui/components/button/Button.tsx

```diff
@@ -7,18 +8,6 @@ import classNames from "@calcom/lib/classNames";
 import Tooltip from "../../v2/core/Tooltip";
 
 export type ButtonBaseProps = {
-  /* Primary: Signals most important actions at any given point in the application.
-       Secondary: Gives visual weight to actions that are important
```

Comment:
All of the variants of the button are automatically inferred by the CVA types down below ✌️